stdlib/public/core/ContiguousArray.swift

```diff
@@ -1037,6 +1038,7 @@ extension ContiguousArray: CustomStringConvertible, CustomDebugStringConvertible
   /// A textual representation of the array and its elements, suitable for
   /// debugging.
   public var debugDescription: String {
+    // Always show sugared representation for Arrays.
```

Comment:
Was it intentional have both changes to /// comments that become documentation and // comments that are only visible in source mixed in the same PR?

Reply:
It's just copy-paste from `Array.swift`.
I believe when this comment was added to `Array.swift` it was just forgotten to add this change also to `ContiguousArray.swift`.